snippets/core/system-text-json/csharp/ImmutablePoint.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace SystemTextJsonSamples
+{
+    // <SnippetImmutablePoint>
```

Comment:
What is the purpose of such comments?

Reply:
This is how we mark sections of code for showing in the doc.  Each snippet is defined by putting a start tag like this before it and an end tag like `// </SnippetNameOfSnippet>` after it.  If a code file doesn't have any of those tags, either the entire file is shown in the doc as a  single snippet or it's not used in the doc at all.